src/pages/study-room/components/timer.test.tsx

```diff
@@ -0,0 +1,46 @@
+import { fireEvent, screen } from '@testing-library/react';
+import { render } from '@/test-helper';
+import { STUDY_ROOM_STEP } from '@/constant/study-room';
+import { StepInfo } from '@/types/study-room.types';
+import Timer from './timer';
+
+const context = describe;
+
+const mockStepInfo: StepInfo = {
+  step: 'PLANNING',
+  progressTime: 10,
+  updateAt: '2024-03-27T05:47:11.638Z',
+};
+
+const mockUseTimer = {
+  remainingSeconds: 600, // 10분
```

Comment:
사소하지만 시간은 요런식으로 표현해줘도 좋을 것 같네요~
```suggestion
  remainingSeconds: 60 * 10,
```